cypress/integration/views/podView.ts

```diff
@@ -0,0 +1,34 @@
+import { flowArguments } from "../../../src/helpers/tests/flow-chart.testData"
+
+describe("The Pod View", () => {
+  beforeEach(() => {
+    cy.visit("/#/flow")
+  })
+
+  it("should have a default 'gateway' pod exist", () => {
+    cy.dataName("chart-node-gateway").should("exist")
+  })
+
+  it("should show 'pod-edit-container' when clicking 'gateway' pod label, and hide it when pressing {esc}", () => {
+    cy.dataName("pod-edit-container").should("not.exist")
+    cy.dataName("chart-node-gateway").dblclick()
+    cy.dataName("pod-edit-container")
+      .should("exist")
+      .type("{esc}")
+      .should("not.exist")
+  })
+
+  it("should set default pod's property input as checkbox whichever its type is boolean, and make it checkable", () => {
+    const booleanProperties = flowArguments.pod.filter(
+      (arg) => arg.type === "boolean"
+    )
+    cy.dataName("chart-node-gateway").dblclick()
+    booleanProperties.forEach((prop) => {
+      cy.get(`input[data-name=pod_${prop.name}_property][type=checkbox]`)
+        .as(prop.name)
+        .should("not.be.checked")
+      cy.get(`@${prop.name}`).check({ force: true }).should("be.checked")
```

Comment:
@franquil, did you get an error around element being detached from DOM when you use the checkbox without `force: true` ?

@BastinJafari, in my observation, this detaching problem only happens in Flow page. We should spend some time to get to the bottom of it.

Reply:
@Roshanjossey thanks for your instant reply. I did encounter errors if `{force: true}` is removed. I suppose in this case it's caused by the target element which is not visible (with style `position: fixed`). I'm not sure it's the same for the Flow page. There're some cases according to the [official guide](https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Forcing), the `force` attribute can be very useful.

Reply:
@franquil, I don't think `position: fixed` is the problem here. It's more likely that React updated the DOM and the element that was selected before is not in the current DOM. I could be wrong here.

Anyways, I think it's good to investigate this and understand what's going on.